user-journey-tests/page-objects/page.js

```diff
@@ -84,7 +84,11 @@ class Page {
   }
 
   async selectContinue() {
+    const deviceName = browser.capabilities?.deviceName
     await page.selectElement(this.getContinueButton())
+    if (deviceName === 'iPhone 13') {
```

Comment:
Could define this as a constant somewhere, or even as an array of device names that need the check for security popup (only iphone 13 for now) and you check if the array includes deviceName